### PR TITLE
DOCSUP-8179: remove control characters

### PR DESCRIPTION
**Pull request by lehasm**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation (changelog entry is not required)

Detailed description:
Fixed issues that may break documentation build:
 - Removed non-printable characters from documentation.
 - Changed table indent in source .md files.
 - single.md is removed after build.

### CODE REVIEW

File: docs/en/sql-reference/functions/hash-functions.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 
 ``` text
 ┌─MurmurHash3──────┬─type────────────┐
-│ 6�1�4"S5KT�~~q │ FixedString(16) │
+│ 6�1�4"S5KT�~~q | FixedString(16) │
```
Comment (alexey-milovidov) [inline docs/en/sql-reference/functions/hash-functions.md:445]: This change is incorrect.

Comment (lehasm) [reply docs/en/sql-reference/functions/hash-functions.md:445]: Can we add hex() function to this example?
```
SELECT hex(murmurHash3_128('example_string')) AS MurmurHash3, toTypeName(MurmurHash3) AS type;
┌─MurmurHash3──────────────────────┬─type───┐
│ 368A1A311CB7342253354B548E7E7E71 │ String │
└──────────────────────────────────┴────────┘
```
It seems more clear to me.

Comment (alexey-milovidov) [reply docs/en/sql-reference/functions/hash-functions.md:445]: Yes, let's add hex.

Comment (alexey-milovidov) [reply docs/en/sql-reference/functions/hash-functions.md:445]: FYI the incorrect change is
`│` to `|`

**Comment by lehasm** [comment]
Added removing single.md after build.
(Hope I got right, that this behavior is intended as it was in some previous revisions.)
@alexey-milovidov, take a look please.

**Comment by alexey-milovidov** [comment]
Ok.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**